packages/documentation/.vitepress/config.ts

```diff
@@ -0,0 +1,307 @@
+// using the defineConfig helper will provide TypeScript-powered
+// intellisense for config options
+import { defineConfig } from "vitepress";
+
+// markdown
+import MarkdownItImplicitFigures from "markdown-it-implicit-figures";
+import MarkdownItPlantuml from "markdown-it-plantuml";
+
+export default defineConfig({
+
+  base: "/ui5-cli/stable/", // use this for github pages deployment or remove for CF
```

Comment:
Is this the right url? `ui5-cli`?
Currently it does not appear anywhere. Our docs URLs look like this: https://ui5.github.io/cli/